linux_pipeline/Jenkinsfile_lis_rpm_hyperv

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env groovy
+
+properties ([
+    [$class: 'ParametersDefinitionProperty',
+        parameterDefinitions: [
+        [$class: 'ChoiceParameterDefinition',
+            name: 'DISTRO_VERSIONS',
+            defaultValue: '',
+            description: 'Distros to be tested']
+        ]
+    ]
+])
+
+def supportedDistros = 'centos_6.4,centos_6.5,centos_6.6,centos_6.7,centos_6.8,centos_6.9,centos_7.0,centos_7.1,centos_7.2,centos_7.3,centos_7.4, \
```

Comment:
create an array for each subset of nodes:
def centos_ws2012 = ''
def centos_ws2012r2 = ""

afterwards you put:
def all_centos = centos_ws2012+ ...
def all_ubuntu = ubuntu_ws2012 +..

afterwards

def ws2012 = centos_ws2012 + ubuntu_ws2012 ...